README.md

```diff
@@ -17,14 +17,30 @@ apt install libzstd-dev
 apt install cmake
 
 # default is release build
-mkdir build
-cd build
-cmake ..
-make
+./scripts/build.sh
+```
+
+If you are using an M1 mac, you will need to build this repository inside a linux docker container, as the build depends on
+some linux-specific libraries. You can run the following command to open a shell in a linux docker container with the current
+directory mounted:
 
-# run unit tests
-ctest
 ```
+export PYTH_REPO=`pwd`
```

Comment:
`PYTH_REPO=/path/to/host/pyth-client` might be more clear. If you'd rather to stick with `pwd`, it should be `"$( pwd )"` to handle whitespace and be consistent with the `podman` command below.